.github/workflows/main.yml

```diff
@@ -44,11 +44,13 @@ jobs:
         cd js-extensions
         pnpm init-repo
 
-    - name: Run tests
+    - name: Run mdbook tests
       run: mdbook test
 
-    - name: Build book
-      run: mdbook build
+    - name: Build book + Run JS tests
+      run: |
+        cd js-extensions
+        pnpm test
```

Comment:
Instead, let's add `pnpm test` to the `commit-check` script in `package.json`, then run `pnpm commit-check` instead of `pnpm init-repo`.